bitnami/influxdb/Chart.yaml

```
@@ -24,4 +24,4 @@ name: influxdb
 sources:
   - https://github.com/bitnami/bitnami-docker-influxdb
   - https://www.influxdata.com/products/influxdb-overview/
-version: 4.0.0
+version: 4.1.0
```

Could you please bump the patch version instead?
```suggestion
4.0.1
```

---

Adapted to only bump revision (4.0.1).